libs/core/src/lib/quick-view/quick-view-group-item-content/quick-view-group-item-content.component.ts

```diff
@@ -1,11 +1,28 @@
-import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { AfterViewInit, ChangeDetectionStrategy, Component, ElementRef } from '@angular/core';
 
 @Component({
     selector: 'fd-quick-view-group-item-content',
     templateUrl: './quick-view-group-item-content.component.html',
-    host: {
-        '[class.fd-input]': 'true'
-    },
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class QuickViewGroupItemContentComponent {}
+export class QuickViewGroupItemContentComponent implements AfterViewInit {
+    /** @hidden */
+    constructor(private readonly _elRef: ElementRef) {}
+
+    /** @hidden */
+    ngAfterViewInit(): void {
+        this._bindElementAttributes();
+    }
+
+    /** @hidden */
+    private _bindElementAttributes(): void {
+        const parentId = this._elRef.nativeElement?.parentElement?.id;
+        const id = `${parentId}__content`;
+
+        const element = this._elRef.nativeElement.querySelector('.fd-quick-view__group-item__content-element');
```

Comment:
I understand that we cannot use `@ViewChild(...)` as it won't find any elements inside a container, but we can provide a static field in the directive with the class (without the leading dot), then here use the reference like `.querySelector('.${QuickViewGroupItemContentElementDirective.class}')` (change to the backquotes) to match elements, it will add some more clarity what we're looking for.